tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py

```diff
@@ -118,6 +118,41 @@ def test_resource_deletion_during_pvc_expansion(self, resource_to_delete):
             ), f"Expansion failed for PVC {pvc_obj.name}"
         log.info("PVC expansion was successful on all PVCs")
 
+        log.info("Verifying new size on pods.")
+        for pod_obj in self.pods:
+            if pod_obj.pvc.volume_mode == "Block":
```

Comment:
why do we skip this validation for RBD?

Reply:
This check is done to ensure filesystem resize is completed before running IO. Filesystem resize is not required if volume mode is block. This check will be done for RBD Filesystem volume mode PVC.

Reply:
got you, thanks!